src/app/index.run.js

```diff
@@ -15,17 +15,21 @@
     });
 
     $rootScope.$on('$locationChangeSuccess', function(e) {
-      if (GData.isLogin()) {
+      var firebaseDriver = $injector.get('firebaseDriver');
```

Comment:
Any reason we're using the `$injector` here? I did it for `$state` because really funky stuff starts to happen if you inject it like everything else.